kformat/kproperty.py

```diff
@@ -56,8 +63,10 @@ class AN(KProperty):
     TIME_FORMAT = '%H%M%S%f'
     TIME_FORMAT_SLICE = 0, -2
 
-    def __init__(self, length: int, *, filler: Optional[bytes] = None) -> None:
+    def __init__(self, length: int, *, filler: Optional[bytes] = None,
+                 errors: Optional[Errors] = Errors.STRICT) -> None:
         super().__init__(length, AN.EXPECTED_TYPES, filler or AN.FILLER)
+        self.errors = errors.value
```

Comment:
```suggestion
        self.errors = errors
```

`.value`로 문자열을 가지고 있을 필요는 없을 것 같습니다. `Enum` 값 자체로 저장하고 이후에 `.value`로 변경해 활용하는 게 더 좋을 것 같습니다 😃

Reply:
UnicodeErrorHandler를 str enum으로 정의하고, 생성자 안에서는 errors Enum으로 가지고 있도록 수정했습니다!